recipes-debian/vlan/vlan_debian.bb

```diff
@@ -0,0 +1,25 @@
+# base recipe: meta-openembedded/meta-networking/recipes-connectivity/vlan/vlan_1.9.bb
+# base branch: warrior
+# base commit: a24acf94d48d635eca668ea34598c6e5c857e3f8
+
+inherit debian-package
+require recipes-debian/sources/vlan.inc
+
+SUMMARY = "VLAN provides vconfig utility"
+HOMEPAGE = "http://www.candelatech.com/~greear/vlan.html"
```

Comment:
This URL is now invalid now. Instead, please refer to the following.
https://salsa.debian.org/debian/vlan